packages/react-vapor/src/components/datePicker/DatePickerSelectors.ts

```diff
@@ -12,11 +14,23 @@ const getDatePicker = (state: IReactVaporState, props: {id: string}): IDatePicke
 };
 
 const getDatePickerLimits = (state: IReactVaporState, props: {id: string}): [Date, Date?] => {
-    const picker = DatePickerSelectors.getDatePicker(state, props);
+    const picker = getDatePicker(state, props);
     return picker ? [picker.appliedLowerLimit, picker.appliedUpperLimit] : [null, null];
 };
 
+const getDatePickerSelected = () =>

```

Comment:
The naming convention for the selectors is usually something like
- `getSomething` --> selector
- `makeGetSomething` or `createSomethingSelector --> selector creator